tests/shared/DevelopmentTestDefs.js

```diff
@@ -48,9 +48,19 @@ gpii.tests.development.testDefs = [{
     },
     gradeNames: ["gpii.test.common.testCaseHolder"],
     gpiiKey: gpii.tests.development.gpiiKey,
-
     sequence: [{
-        func: "{loginRequest}.send"
+        // TODO: Wait till the noUser keyin at the GPII start completes. This is to work around an issue with
```

Comment:
We need to solve this by boiling the noUser logon event into a new onReady event for the whole server. This 100ms timeout will be unstable and unreliable.

Reply:
Why does this issue not affect all acceptance tests?

Reply:
Acceptance tests need to start and load data into a pouchdb express server, which takes some time and completes much later than noUser keyin process. The test sequence runs when all servers to be ready at when noUser has already keyed in.

The production config test only needs to start the local flow manager to talk to a running GPII cloud. This process doesn't include data loading etc so the LFM server gets ready sooner than the noUser keyin completes.

I've improved the production config test to start the test sequence when a new aggregate event "onAllReady" is fired.  "onAllReady" event listens to both "onServerReady" and "{flowManager}.events.noUserLoggedIn".